configure_workers.xsh

```diff
@@ -0,0 +1,32 @@
+                                          'script_number': '00'},
+                  'cf-auto-tick': {'doctr_key': 'SiGX99CpSLIHQXcOR8RN3JfL9pdyps1kFISSaaICFOJ65Wp+C5ERWzqZMOr7NlmTijJmCQ4DzGi9h1t+zYWgg+l2XsUXLS+4SDI5SZMCmbP5vHiP2qxnWEiCPTaZrXMNeGcD4yW1KvwxnrRIcyX4JDz2EP4cLKS0laj3kpGTrYPVx2wFrX8/V78SPIZ9S2m1aSOmOtEvyaj1gOdazxGXjLyZ43FbrhwKILweVd8wgsKS4GKsgQIXDRxk4713f32SMOzav+xI6IzHxaoo3RQu3NQQfoeks73/hfxdNyr76fggimXKUPo5tS84W5rU6fwhhZU/ojSxJIMnGcZSRGpmk86nd3l6+jSqU6txinwO5ROL3wvu0o06cY17WAUzPfF/z5Oo5sNbWeuj9Idp9ske4wfWx17q0Z/jkOBiRXYGetjdQTwRlRrqf9R7hja6QGm0giSgXTMGNMyWsI+YIWAZcl5vYBuPCFGc8RUfuRz4zL34oN/4pa7rewkamkNlgTxx9V+lBj46+tNMlrhWhF97z+p+Bpl1zauGxCxCSEGxeWSNyIHgu4z4CKm9q4LG5G201QC71P+6i9Zbo0xcX5PnZi/V1Smky8yQJ4F791GJfSsCMfo98FIQJ7ZoLFyB2YP2dLH538QXL51IKGz8E8PMzZKDpVi+bOzj6iN7e9Qw0LM=',
+                                   'script_number': '01'},
+                  'graph-upstream': {'doctr_key': '1ZYymeT9HNKhWGEb6aZ6oxSd2zOYD1L1S0OcWZ8AlWwmOQ7xqBbZR9htT3oA4fl6AoLbFHwaL6v/Y/WfstKOpm9xjUtigFNKsJiAEUp/C/TxeSJen8bvGyT3ja6xxIhJmz7LrwN6ycXPvYoPqXE0gvCAN44IIWRRIaBOa7Q7igeY2u5iSyv4DaFpe0LycdKxvkV9hm9mZKKFOFZ+UCBhiosB76bSXJ49zYucSgn6MUnzpYVTdlvGko85powav0OMXjDs2Lwju+sL9D+5vB49gbeY9jPMv4cVrOSm2CPql7sb7WwlEl3JHxCjLXRgVu1sWRkH/2GGj0Mbu2u6gjxbJ6btGG0p8Q0VROLk+hV9VlVdxFImlsQNL2a+e1MgHZg92ZNM57E8qGASlBG4cRG7qWDi1AQY0e2AiwVJVRtCuc7kYzcFTQ80xqwiuV9cYQDTlKgQdn+esXs+t57yOBYU013ydg6s6JRhu9utnmBjMfvov+NQ37ECohLxPWJaLORJozDWLBLfDr+5PbN4oqzN0V/wKCIg1vXGDKc8L4gH1k/m10BfUwpsyTsXgyupDpIYldyryNU5NKmn5Jq6YqfpBwGDYFeYFUrvDwpwRMPg81zmbJSindKr/oRJ0g3bNFv7nOOd0zxF3HE4WPJADtW1CbweJxpx8LbneC1uvM3FxOQ=',
+                                     'script_number': '02'},
+                  'make-cf-graph': {'doctr_key': 'kUlHJeVjxNf/dt9XSSV+mgHH/9qp30zqABjwdeDe1Nbs5JPVCuLjD7yqL/Pln7jQdzGaTksEIUoFggGSKpXi1SCIZCOlzBax/wAIPTSvJEvUIzFs6Vws7J/mV147tyoNanOHvNw6CvicNhEuWD8HCWUw36kF93yhePwuFvNRWgIof8Ua54d3l2sGTCEHSbIiSvx6eF5WVsIKOWVyjKGu5H3y6HSoOGkFlezw+d9Y2owkx99KPKgDRFRt1BCuKxjhi8Yuv5HTKzCwDuQFC6Ak8uriVTQfMv44Tssl3SP4bDu+1x+zr7mMHTTOHqlYFSOlBtSkDAf6Cctfk903kFQI5TkiDLCSe86tlUiiC2qcFtgL4kll4SMQ5Eu1nmoCjQk457KUb5nkIpWFS5g/TxM8qy0CZF/KHsz2n9M76iDg8V2qS5xyb2hglt51HCGb94tnuR6oBPx+yDzBTGm38c4I8vpzT+8uDEr9bbjh7pO0hzssNCSc/XZld8ctcA5l1M+9Ib/W7QwqK0uelh2VEwpnd8JeMEXGw5jaT+1EA3FUnNTBBOGzJXK3SzG8LAJdQjChSl8vdDvuug2VN04433oZ+PhWv9YrPc+XBs++X/re4pYK7t1dXFBx7qlyMFRWTCYhiPhzgBwbeojuB/MuL6jGld9UqMews+NjdFIyY3j0W6o=',
+                                    'script_number': '03'}}
+
+r = '/home/christopher/dev/'
```

Comment:
We might want to make this a command line argument, or perhaps an environment variable.